### PR TITLE
feat: added django example

### PR DESCRIPTION
**Pull request by gen1us2k**

### CODE REVIEW

**Comment by gen1us2k** [comment]
That's a basic example for upcoming blogpost 'Securing Django apps using Ory Cloud'

File: django-ory-cloud/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+This repo demonstrates how you can use Ory Cloud or Ory Kratos with Django apps.
+This app is not for production use and serves as an example of integration.
+It was generated using the default `django-admin startproject` command. It uses sqlite3 as the database backend.
+Consider using [django-cookiecutter](https://github.com/cookiecutter/cookiecutter-django)
+
+## Prerequisites
+
+1. Python 3.10
+2. Poetry
```
Comment (vinckr) [inline django-ory-cloud/README.md]:
```suggestion
2. [Poetry](https://python-poetry.org/) - Python dependency manager.
```

File: django-ory-cloud/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+3. Docker (in case you use Ory Kratos self-hosted)
```
Comment (vinckr) [inline django-ory-cloud/README.md]:
```suggestion
3. Docker (if you want to  self-host Ory Kratos)
```

File: django-ory-cloud/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+   poetry run python manage.py migrate
+   poetry run python manage.py runserver
+
+```
+
+## Environment variables
+
+```
+   ORY_SDK_URL=https://playground.projects.oryapis.com
+   ORY_UI_URL=https://playground.projects.oryapis.com/ui
```
Comment (vinckr) [inline django-ory-cloud/README.md]:
```suggestion
export ORY_SDK_URL=https://playground.projects.oryapis.com
export ORY_UI_URL=https://playground.projects.oryapis.com/ui
```

**Review by vinckr — DISMISSED** [review]
Neat!
We can do more stuff with this, good start 🙌

**Review by vinckr — APPROVED** [review]

**Pull request merged**